charts/hlf-k8s/templates/deployment-application-channel-operator.yaml

```diff
@@ -108,7 +108,7 @@ spec:
                 ## Check if update has been already sent
                 printf "[DEBUG] Check if $org is in the application channel \n"
                 configtxlator proto_decode --input channel.block --type common.Block | jq .data.data[0].payload.data.config > channel-$org-hasjoined.json
-                if grep "$org" channel-$org-hasjoined.json > /dev/null; then
+                if grep "${org}MSP" channel-$org-hasjoined.json > /dev/null; then
```

Comment:
This assumes that the organisation ID is name+"MSP" which could not be.

Reply:
Yes, it a quick fixup, we should update 
```yaml
systemChannel:
  name: systemchannel
  organizations:
    ## The organizations to add to the system channel
    #   - { org: MyPeer1, configUrl: peer-1-org-config-operator.org-1/config/configOrg.json }
    #   - { org: MyPeer2, configUrl: peer-2-org-config-operator.org-2/config/configOrg.json }

appChannel:
  name: mychannel
  ## The organizations to add to the application channel (see README)
  # organizations:
  #   - { org: Org1, configUrl: peer-1-org-config-operator.org-1/config/configOrgWithAnchors.json }
  #   - { org: Org2, configUrl: peer-2-org-config-operator.org-2/config/configOrgWithAnchors.json }
  ## The organizations to fetch signed proposals from (see README)
  # proposalOrganizations:
  #   - { org: Org1, proposalServerUrl: peer-1-org-application-channel-operator.org-1/proposal/ }
  #   - { org: Org2, proposalServerUrl: peer-2-org-application-channel-operator.org-2/proposal/ }
```

to 
```yaml
systemChannel:
  name: systemchannel
  organizations:
    ## The organizations to add to the system channel
    #   - { org: MyPeer1, mspid: MyPeer1MSP, configUrl: peer-1-org-config-operator.org-1/config/configOrg.json }
    #   - { org: MyPeer2, mspid: MyPeer2MSP, configUrl: peer-2-org-config-operator.org-2/config/configOrg.json }

appChannel:
  name: mychannel
  ## The organizations to add to the application channel (see README)
  # organizations:
  #   - { org: Org1, mspid: Org1MSP, configUrl: peer-1-org-config-operator.org-1/config/configOrgWithAnchors.json }
  #   - { org: Org2, mspid: Org2MSP, configUrl: peer-2-org-config-operator.org-2/config/configOrgWithAnchors.json }
  ## The organizations to fetch signed proposals from (see README)
  # proposalOrganizations:
  #   - { org: Org1, mspid: Org1MSP, proposalServerUrl: peer-1-org-application-channel-operator.org-1/proposal/ }
  #   - { org: Org2, mspid: Org2MSP, proposalServerUrl: peer-2-org-application-channel-operator.org-2/proposal/ }
```
And configmaps associated